cli/tests/073_worker_error.ts.out

```diff
@@ -2,4 +2,6 @@
     at foo ([WILDCARD])
     at [WILDCARD]
 error: Uncaught (in promise) Error: Unhandled error event reached main worker.
+              throw new Error("Unhandled error event reached main worker.");
+                    ^
```

Comment:
Some output has more left padding than the others (eg. `059_fs_relative_path_perm.ts.out`). Is that intentional?

Reply:
This is not padding (that is always 8 spaces), this is just the indentation before the statement in the file.